frontend/homework/11/src/components/Filter/Filter.scss

```diff
@@ -0,0 +1,43 @@
+$color-accent: #ee99c2;
+
+.filters {
+  display: flex;
+  padding: 20px;
+  align-items: flex-end;
+    overflow: hidden;
+    border-bottom: 1px solid black ;
+  }
+  
+.tags {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+    list-style: none;
+    padding: 0;
+  }
+
+  .taged {
```

Comment:
could have been named better denoting the actual logic and following BEM convention